.github/workflows/inductor-rocm.yml

```diff
@@ -0,0 +1,47 @@
+name: inductor-rocm
+
+on:
+  schedule:
+    # We have several schedules so jobs can check github.event.schedule to activate only for a fraction of the runs.
+    # Also run less frequently on weekends.
+    - cron: 45 0,4,8,12,16,20 * * 1-5
+    - cron: 45 4,12 * * 0,6
```

Comment:
@huydhn I consolidated the schedules, but would prefer to keep the weekends lighter to not strain the CI unnecessarily.